.github/workflows/acceptance-tests.yml

```diff
@@ -478,4 +481,41 @@ jobs:
           PARALLEL_GO_TEST: 20
           CI: true
           TEST_REGEX: "^TestAccSTSAssumeRole"
+        run: make testacc
+
+  event_trigger:
+    needs: [ change-detection ]
+    if: ${{ needs.change-detection.outputs.config == 'true' || github.event_name == 'workflow_dispatch' || github.event_name == 'schedule' || github.event.label.name == 'run-testacc' || github.event.label.name == 'run-testacc-config' || inputs.parent-event-name == 'release' }}
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v4
+      - name: Set up Go
+        uses: actions/setup-go@v4
+        with:
+          go-version-file: 'go.mod'
+      - name: Acceptance Tests
+        env:
+          MONGODB_ATLAS_BASE_URL: ${{ vars.MONGODB_ATLAS_BASE_URL }}
+          MONGODB_REALM_BASE_URL: ${{ vars.REALM_BASE_URL }}
+          MONGODB_ATLAS_PUBLIC_KEY: ${{ secrets.MONGODB_ATLAS_PUBLIC_KEY_CLOUD_DEV }}
+          MONGODB_ATLAS_PRIVATE_KEY: ${{ secrets.MONGODB_ATLAS_PRIVATE_KEY_CLOUD_DEV }}
+          MONGODB_ATLAS_PROJECT_OWNER_ID: ${{ vars.MONGODB_ATLAS_PROJECT_OWNER_ID }}
+          MONGODB_ATLAS_ORG_ID: ${{ vars.MONGODB_ATLAS_ORG_ID_CLOUD_DEV_NETWORK }}
+          MONGODB_ATLAS_PROJECT_ID: ${{ vars.MONGODB_ATLAS_ACCTEST_TRIGGER_PROJECT_ID_CLOUD_DEV }}
```

Comment:
LGTM. 
I am not fully aware of manual steps needed to set up project for trigger but was curious if it requires having a cluster running. If this is the case, are we sure the dev environment does not remove the created cluster after some time?

Reply:
good question - it is not needed to have a cluster to run these type of tests. What is is needed though are:
- a project
- its related app
- a function
That's why I created a project and put it as DO_NOT_DELETE [here](https://github.com/mongodb/terraform-provider-mongodbatlas/pull/1573) so all its related resource (app and function) should be preserved.